packages/manager/apps/key-management-service/src/pages/listing/terminate/TerminateKms.tsx

```diff
@@ -13,8 +15,20 @@ export default function TerminateKms() {
 
   const { terminateKms, isPending } = useTerminateOKms({
     okmsId,
-    onSuccess: closeModal,
-    onError: closeModal,
+    onSuccess: () => {
+      trackPage({
+        pageType: PageType.bannerSuccess,
+        pageName: 'delete_kms_success',
+      });
+      closeModal();
+    },
+    onError: () => {
+      trackPage({
+        pageType: PageType.bannerError,
+        pageName: 'delete_kms_error',
+      });
+      closeModal();
```

Comment:
You also close the modal on error ?
You don't display the error on the modal ?

Reply:
Ee handled error on the previous page (dashboard, list).
In any case, this modal will be replaced by the react-manager-component `<DeleteModal />`, so we'll fixed that.